setup.py

```diff
@@ -14,7 +14,7 @@ def readme():
     with open('README.rst') as f:
         return f.read()
 
-requiredpackages = ['pyrpn', 'h5py', 'numpy', 'matplotlib', 'pyepics']
+requiredpackages = ['pyrpn', 'h5py', 'numpy', 'matplotlib', 'pyepics', 'wxPython']
```

Comment:
The only concern about adding `wxPython` as one of the deps is the compatibility since pip can only install 4.x version, which is not working for the present codebase.

Reply:
Good point. Probably it is better to mention the dependency on wxPython in the README and leave it out here.